ansible/Makefile

```diff
@@ -29,7 +29,7 @@ provision-vms:
 
 .PHONY: destroy-vms
 destroy-vms:
-	ansible-playbook -i $(CONTEXT) \
+	ansible-playbook -i $(CONTEXT) -vvvv \
```

Comment:
I did this because of a weird flake I got, but then the issue never happened again and I forgot to remove the verbosity, I'll roll it back before merging.